autogen.sh

```diff
@@ -1,168 +0,0 @@
-#! /bin/sh
-# Run this to generate all the initial makefiles, etc.
-
-# Stolen from the GNU Midnight Commander. Customized for giFTcurs.
-# adapted for xqf
-
-# Make it possible to specify path in the environment
-: ${AUTOCONF=autoconf}
-: ${AUTOHEADER=autoheader}
-: ${AUTOMAKE=automake}
-: ${ACLOCAL=aclocal}
-: ${GETTEXTIZE=gettextize}
-
-srcdir=`dirname $0`
-test -z "$srcdir" && srcdir=.
-
-(
-cd $srcdir
-
-# For autotools compatibility
-if ! test -e README; then
-	ln -s README.md README
-fi
-
-# For autotools compatibility
-if ! test -e NEWS; then
-	ln -s NEWS.md NEWS
-fi
-
-# For autotools compatibility
-if ! test -e AUTHORS; then
-	ln -s AUTHORS.md AUTHORS
-fi
-
-# extract flag icons
-rm pixmaps/flags/*.png
```

Comment:
Here the flags were deleted if they are already there (not expected to be in the repository).